external/dynamo/start_dynamo_unified_thompson_hints.sh

```
@@ -61,8 +61,9 @@ LOCAL_MODEL_DIR="${DYNAMO_MODEL_DIR}"
 
 # Repository directory - auto-detect from script location or use env var
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-LOCAL_REPO_DIR="${DYNAMO_REPO_DIR:-$(cd "$SCRIPT_DIR/../.." && pwd)}"
-CUSTOM_DYNAMO_DIR="$LOCAL_REPO_DIR/external/dynamo/generalized"
+LOCAL_REPO_DIR="${DYNAMO_REPO_DIR:-$SCRIPT_DIR}"
```

Shellcheck (0.11.0)

[warning] LOCAL_REPO_DIR appears unused. Verify use (or export if used externally).

(SC2034)